packages/prices/src/prices.ts

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable eqeqeq */
+import { PriceSpecification } from "@js-ligo/vocab";
+import { LigoAgreementState } from "@js-ligo/vocab";
+import { RentalCarReservation } from "@js-ligo/vocab";
+import { Offer } from "@js-ligo/vocab";
+
+export class Prices {
+  /**
+   * Find Total Price
+  // Case 1: Base Price per day
+  // Case 2: Base price per kilometer
+  // Case 3: Base price per day + X$ per kilometer over Y range
+  // Case 4: Base price per hour
+  // Case 5: Different price per day
+  // Case 6: Monthly Subscription of X$
+  // Case 7: Discount for Y+ days
+  */
+  async CalculateTotalPrice(
+    _priceSpecifications: Offer["priceSpecifications"],
```

Comment:
Can this type just be `PriceSpecification[]`?

Reply:
I couldn't mention `PriceSpecification[ ]` only. Offer has `priceSpecifications` with `PriceSpecification[ ]` so I mentioned it this way.

Reply:
It seems that it showed error as it was unused in the beginning.